manifest.tmpl

```diff
@@ -0,0 +1,12 @@
+image: rancher/machine:{{#if build.tag}}{{build.tag}}{{else}}{{replace "release/" "" build.branch }}-head{{/if}}
+manifests:
+  -
+    image: rancher/machine:{{#if build.tag}}{{build.tag}}{{else}}{{replace "release/" "" build.branch }}-{{build.commit}}{{/if}}-linux-amd64
+    platform:
+      architecture: amd64
+      os: linux
+  -
+    image: rancher/machine:{{#if build.tag}}{{build.tag}}{{else}}{{replace "release/" "" build.branch }}-{{build.commit}}{{/if}}-linux-arm64
+    platform:
+      architecture: arm64
+      os: linux
```

Comment:
Is this needed ? I haven't seen one before, thought it gets created by plugin typically.

Reply:
From what I can tell, there are two ways to do this: (1) use this template file, or (2) do some sort of pattern matching of the `REPO/IMAGE:TAG` (this is how rancher-operator works).

I pulled the manifest.tmpl file from rancher/rancher.

Reply:
The operator way looks much simpler, if we can get away with that it would be better.